### PR TITLE
Lesson 3

### PR DESCRIPTION
**Pull request by jakedaylong**
Slicing Lab and List Lab completed and submitted.

### CODE REVIEW

File: students/jacob_daylong/Lesson 3/List_Lab/List.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+print(fruits_series_1)
+
+new_fruit = input("Please add another fruit: ")
+
+fruits_series_1.append(new_fruit)
+
+print(fruits_series_1)
+
+fruit_num = int(input("Please select a fruit by number: "))
```
Comment (charlesrobison) [inline students/jacob_daylong/Lesson 3/List_Lab/List.py:58]: len() is nice here to show how many items are in the list

File: students/jacob_daylong/Lesson 3/List_Lab/List.py [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+print()
+print("Add Watermellon:")
+fruits_series_1.insert(0, "Watermellon")
+print(fruits_series_1)
+print()
+
+print("Fruits starting with 'P'")
+for i in fruits_series_1:
+    if i[0] == "P":
```
Comment (charlesrobison) [inline students/jacob_daylong/Lesson 3/List_Lab/List.py:76]: Good idea to user upper() or something similar to catch case sensitive user input

File: students/jacob_daylong/Lesson 3/List_Lab/List.py [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+print(fruits_series_1)
+fruits_series_3 = fruits_series_1[:]
+fruit_ans = ""
+
+for i in fruits_series_3[:]:
+    fruit_ans = input("Do you like " + i.lower() + " ? ")
+    while fruit_ans not in ['yes', 'no']:
+        fruit_ans = input ("Please input yes or no. ")
+    if fruit_ans == 'no':
+        fruits_series_3.remove(i)
```
Comment (charlesrobison) [inline students/jacob_daylong/Lesson 3/List_Lab/List.py:104]: Nice!

**Review by charlesrobison — APPROVED** [review]
Looks good!  Just a couple of suggestions.  Make sure you submit your code on the course site for grading.

Ready to merge

**Comment by natasha-aleksandrova** [comment]
@jakedaylong please remove .gitignore file - you should not be adding any files outside of your student directory.

you can add a .gitignore file in your own folder

**Comment by natasha-aleksandrova** [comment]
@jakedaylong thank you for the cleanup!

**Pull request merged**